tools/shaderc/shaderc_spirv.cpp

```diff
@@ -219,6 +222,118 @@ namespace bgfx { namespace spirv
 		}
 	}
 
+	bgfx::TextureFormat::Enum SpirvImageFormatToTextureFormat(spv::ImageFormat format) {
```

Comment:
These `spv::ImageFormat::ImageFormat*` are all contigious, you could create remap table instead of crazy swtich.

Reply:
Done